10.9-libcxx/stable/main/finkinfo/libs/perlmods/yaml-tiny-pm.info

```diff
@@ -3,14 +3,11 @@ Package: yaml-tiny-pm
 Version: 1.73
 Revision: 1
 Description: Read/Write YAML files
-Distribution: 10.10, 10.11, 10.12, 10.13
-Type: perl, systemperl (5.18.2)
+Type: perl
 License: Artistic/GPL
 Maintainer: Benjamin Reed <yaml-tiny-pm@fink.raccoonfink.com>
 BuildDepends: <<
-	fink (>= 0.30.2),
-	module-build-tiny-pm%type_pkg[systemperl] (>= 0.027),
-	perl%type_pkg[systemperl]-core
+	fink (>= 0.30.2)
```

Comment:
Can probably even drop the BDep:fink (remnant of DefaultScript?) and the Info2: wrapper (remnant of varianting and part of RangerRick's template).

Reply:
You're welcome to drop the BDep:fink and Info2 wrapper (you're right that I borrowed them).